clusters/build-clusters/01_cluster/openshift-monitoring/main_alertmanager.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: monitoring.coreos.com/v1
+kind: Alertmanager
+metadata:
+  labels:
+    alertmanager: main
+  name: main
+  namespace: openshift-monitoring
+spec:
+  affinity:
+    podAntiAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+      - podAffinityTerm:
+          labelSelector:
+            matchExpressions:
+            - key: alertmanager
+              operator: In
+              values:
+              - main
+          namespaces:
+          - openshift-monitoring
+          topologyKey: kubernetes.io/hostname
+        weight: 100
+  baseImage: openshift/prometheus-alertmanager
+  containers:
+  - args:
+    - -provider=openshift
+    - -https-address=:9095
+    - -http-address=
+    - -email-domain=*
+    - -upstream=http://localhost:9093
+    - '-openshift-sar={"resource": "namespaces", "verb": "get"}'
+    - '-openshift-delegate-urls={"/": {"resource": "namespaces", "verb": "get"}}'
+    - -tls-cert=/etc/tls/private/tls.crt
+    - -tls-key=/etc/tls/private/tls.key
+    - -client-secret-file=/var/run/secrets/kubernetes.io/serviceaccount/token
+    - -cookie-secret-file=/etc/proxy/secrets/session_secret
+    - -openshift-service-account=alertmanager-main
+    - -openshift-ca=/etc/pki/tls/cert.pem
+    - -openshift-ca=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+    - -skip-auth-regex=^/metrics
+    env:
+    - name: HTTP_PROXY
+    - name: HTTPS_PROXY
+    - name: NO_PROXY
+    image: quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:6d952943885b8c9d972db55153206cb8ef5ee8e1d3aa4f76dec0cc616b31b4cc
```

Comment:
What is this? Not sure why we're referring to the image this way but `v4.0` sounds old-ish.

Reply:
It is not me to decide the image. It is the installer of the cluster.

My guess: we installed a nightly build (that was the decision at that time ... we want to test the most recent and greatest) ... which might cause the informal tag ... but i have no idea why it sounds 4.0 ish.
```
$ oc get clusterversion
NAME      VERSION                             AVAILABLE   PROGRESSING   SINCE   STATUS
version   4.2.0-0.nightly-2019-10-11-230724   True        False         20d     Cluster version is 4.2.0-0.nightly-2019-10-11-230724
```

Reply:
ok
/shrug

Reply:
The problem is hidden now by not exposing the CR `prometheus` and `alertmanager`.